crates/fluvio/Cargo.toml

```diff
@@ -78,7 +78,7 @@ async-std = { version = "1.6.4", default-features = false }
 fluvio-future = { version = "0.3.0", features = ["io", "fixture"] }
 
 [build-dependencies]
-built = "0.5.1"
+built = { version = "0.5.1", features = ["git2"] }
```

Comment:
I am a bit concerned that this adds more compile-time since `git2` seems to depend on a bunch of other dependencies.  Maybe just do similar way `cli` crates?

Reply:
Changed to be similar to `cli` approach